validation_run.py

```diff
@@ -0,0 +1,182 @@
+# This code is used to generate simulations using parameters from the single outbreaks. Each set of parameters will generate 5000 valid results (We only consider the simulations with number of introductions on day 1 > 0)
```

Comment:
Suggest changing file name to be more descriptive, something like - single_outbreak_validation

Reply:
sounds good